src/core_modules/capture-core/components/FormFields/New/HOC/withLabel.js

```diff
@@ -16,8 +17,12 @@ const getStyles = (theme: Theme) => ({
 
 const getStylesLabel = (theme: Theme) => ({
     container: {
-        display: 'flex',
-        alignItems: 'center',
```

Comment:
I think the bit that will make this work is to have this container back, and wrap the whole `Tooltip`.

Can you please try that if you got the time Eirik?